src/run_integ_test.py

```diff
@@ -5,60 +5,40 @@
 # this file be licensed under the Apache-2.0 license or a
 # compatible open source license.
 
-import logging
 import os
 import sys
 
-from manifests.build_manifest import BuildManifest
-from manifests.bundle_manifest import BundleManifest
 from manifests.test_manifest import TestManifest
 from system import console
-from system.temporary_directory import TemporaryDirectory
-from test_workflow.dependency_installer_opensearch import DependencyInstallerOpenSearch
-from test_workflow.integ_test.integ_test_suite_opensearch import IntegTestSuiteOpenSearch
+from test_workflow.integ_test.integ_test_runner_opensearch import IntegTestRunnerOpenSearch
+from test_workflow.integ_test.integ_test_runner_opensearch_dashboards import IntegTestRunnerOpenSearchDashboards
 from test_workflow.test_args import TestArgs
-from test_workflow.test_recorder.test_recorder import TestRecorder
-from test_workflow.test_result.test_suite_results import TestSuiteResults
 
 
 def main():
     args = TestArgs()
+
     console.configure(level=args.logging_level)
-    test_manifest_path = os.path.join(os.path.dirname(__file__), "test_workflow", "config", "opensearch", "test_manifest.yml")
+
+    test_manifest_path = os.path.join(
```

Comment:
I think you don't need this if the caller always has to specify a test manifest, which is completely fine (make that change).